lib/encoder/include/Pololu37D.h

```diff
@@ -22,8 +24,20 @@ class Pololu37D final : public Encoder {
   bool reset() override;
 
  private:
-  GPIO::QEI m_QEI;
+  bool read();
+  float return_delta_time();
```

Comment:
Remove this function, it's not getting used anywhere

Reply:
done